README.md

```diff
@@ -54,15 +54,18 @@ vars:
     using_invoice_bundle: false         #disable if you don't have invoice bundles in QuickBooks
     using_journal_entry:  false         #disable if you don't have journal entries in QuickBooks
     using_payment:        false         #disable if you don't have payments in QuickBooks
+    using_purchase_order: false         #disable if you don't have purchase orders in QuickBooks
     using_refund_receipt: false         #disable if you don't have refund receipts in QuickBooks
     using_transfer:       false         #disable if you don't have transfers in QuickBooks
     using_vendor_credit:  false         #disable if you don't have vendor credits in QuickBooks
+    using_sales_receipt:  false         #disable if you don't have sales receipts in QuickBooks
+    using_purchase_order: true          #enable if you want to include purchase orders in your staging models
```

Comment:
duplicated

Reply:
I actually think the top one is duplicated.  The `using_purchase_order` variable should be set to `false` by default, so our instructions should show users how to do the opposite.

Reply:
Removed the duplicate. `using_purchase_order` is by default set to `false`. The readme now shows to enable as `true` if it is to be used.